docs/modules/ROOT/pages/how-to-guides/Configure-components/proc_reviewing_configging_components.adoc

```diff
@@ -0,0 +1,27 @@
+//[id="proc_reviewing_configging_components_{context}"]
+
+= Reviewing and configuring components
+
+[role="_abstract"]
+If you imported your own code, from the **Review and configure for deployment** pane, make sure to define, review, or change your application and component settings before you finish importing. You can rename a component here, too. 
+
+. Make sure that **Component name** is correct. Click the GitHub link under this field if you want to check your repository URL.
+. Confirm that {ProductName} detected the right **Runtime**. If so, you should notice that the suggested deployment options are appropriate. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
+
+In the **Build & deploy configuration** section, look over these fields:
+
+[start=3]
+. **Target port**: Is it correct? If not, click in the field to modify it.
+. **Dockerfile**: If you specified **Dockerfile** as your runtime, make sure {ProductName} detected the right one. Click in the field if you need to modify it. Don't worry! {ProductName} hides this field if you chose a different runtime.
```

Comment:
@christianvogt I know we had discussed trying to hide the Dockerfile field, but aren't we always showing it? I haven't seen it hide in the current implementation. Sometimes there's a remote value that is displayed (like when I try to build https://github.com/mattreid/partner-catalog2 which detects as Spring Boot).

Reply:
thanks, @christianvogt. is it not hidden when a user doesn't choose it as a runtime?